healthcheck.go

```diff
@@ -0,0 +1,13 @@
+package runhttp
+
+import "net/http"
+
+// HealthCheckHandler responds with a 200, as required by services under Micros
```

Comment:
NIT: since this is in the open source repo, we generally don't want to mention internal Atlassian things like "Micros", so the second part of this comment should probably be omitted.